README.md

```diff
@@ -1 +1,64 @@
-# electric-eagles
+# Mercury
+Mercury is a MLH Externship Fellowship project by the Electric Eagles group from Pod 2-2-1.
+
+## About
+
+This purpose of this project is to build a communication platform that eases the common paperwork burden for the reserve units in the US Army. Soldiers in such units often have to fill out tedious paperwork to request permission for absence from weekend drills. COVID-19 also brings about additional paperwork. Unfortunately, all such paperwork would need to be printed, then scanned & uploaded, by soldiers, only to be printed-scanned-signed again by their commanders. Mercury aims to streamline this process by digitizing these steps.
+
+The name Merucy was inspired by Mercurius who, in Roman religion, was the god of communication.
+
+The project uses React Native which is helpful in developing a single application that deploys to both Android and iOS devices. The chat feature is built using the GiftedChat library.
+
+## Features
+
+- [x] Users can login/sign-up (authentication)
+- [x] Users see a scrollable list of upcoming events
+- [x] Users can have access to features & information depending on their rank-based account privileges (authorization)
+- [x] Users can see a COVID screening form
+- [x] Users can view & edit their profiles
+- [x] Users can direct message (1-on-1 chat)
+- [x] Users can message a group (group chat)
+
+## Wiki Documentation
+
+The project Wiki Documentation can be found [here](https://github.com/junqili259/Mercury/wiki).
+
+
+## Tools we used
```

Comment:
For this section, I recommend listing the tools we used like firebase, react native etc.